tasks/poll-index.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * this script polls the Lunie site every 10 seconds and extracts its etag to check if there has been 
+ * any new Lunie versions deployed on Netlify while the user had the app open on the browser
+ * 
+ * Usage:
+ *
+ * node tasks/poll-index.js
+ */
+
+const fetch = require("node-fetch")
```

Comment:
why node fetch?

Reply:
It is the one we are using in the other scripts 🤷‍♀

Reply:
lol yeah but they run in Node.js. This runs in the browser where `fetch` is native

Reply:
oh I see 😮

Reply:
I tried removing it and it didn't work 🤷‍♀ it said `fetch undefined`

Reply:
Ur still executing the script with node...